libs/store_helper/ebpf_store_helper.c

```diff
@@ -0,0 +1,338 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#ifndef USER_MODE
+#include "kernel\ebpf_registry_helper_km.h"
+#else
+#include "user\ebpf_registry_helper_um.h"
+#endif
```

Comment:
if you rename both files to be ebpf_register_helper.h (just like we have two framework.h files) then you don't need the ifdef here.

Reply:
I don't have a strong opinion on this, although we'd need to rename the c/cpp files as well so to have them matching.

Reply:
They don't need to match, in my opinion.  We have ebpf_platform.h, ebpf_platform_user.cpp, and ebpf_platform_kernel.cpp (as opposed to two ebpf_platform.cpp). Not a big deal either way though, but the cpp file names don't matter for purposes of removing ifdefs, though the .h file names do.